src/Content/RichContent.js

```diff
@@ -0,0 +1,31 @@
+import { createElement, Component } from 'react';
+import { object, string } from 'prop-types';
+
+export default class RichContent extends Component {
+    static propTypes = {
+        sanitizedRawHtml: string.isRequired,
+        wrapperTag: string,
+        wrapperProps: object
+    };
+
+    static defaultProps = {
+        wrapperTag: 'span',
+        wrapperProps: {
```

Comment:
It's more common in community React components to just spread any non-own prop into the wrapper in this case. So your `render` would be:

```jsx
render() {
    const { wrapperTag: Tag, ...others } = this.props;
	return (
		<Tag className='peregrine-raw-html' {...others} />
	);
}
```

and you drop `wrapperProps` entirely

Reply:
I'm so accustomed to not using object spread for support reasons that I forgot it was an option. I'll change the API.